### PR TITLE
[RORDEV-317] add es 7.9.0 support

### PR DESCRIPTION
**Pull request by clutroth**
🚀New 7.9.0 support

### CODE REVIEW

**Comment by clutroth** [comment]
es netty api has changed

**Comment by clutroth** [comment]
es netty api has changed. Look on ReadonlyRestPlugin.

File: es79x/src/main/scala/tech/beshu/ror/es/ReadonlyRestPlugin.scala [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+  extends Plugin
+    with ScriptPlugin
+    with ActionPlugin
+    with IngestPlugin
+    with NetworkPlugin
+    with ClusterPlugin {
+
+  LogPluginBuildInfoMessage()
+
+  Constants.FIELDS_ALWAYS_ALLOW.add(IgnoredFieldMapper.NAME)
```
Comment (coutoPL) [inline es79x/src/main/scala/tech/beshu/ror/es/ReadonlyRestPlugin.scala:84]: what about other fields?

File: bin/build.sh [hunk 2]
```diff
@@ -91,6 +91,9 @@ if [[ $TRAVIS != "true" ]] ||  [[ $ROR_TASK == "package_es7xx" ]]; then
 
     echo ">>> ($0) additional builds of ES module for specified ES version"
 
+    #es79
+    ./gradlew --stacktrace es79x:ror '-PesVersion=7.9.0'
```
Comment (coutoPL) [inline bin/build.sh:95]: cannot find files:
* build.gradle
* gradle.properties

in es79x module

File: core/src/main/scala/tech/beshu/ror/utils/SetOnce.scala [hunk 3]
```diff
@@ -0,0 +1,9 @@
+package tech.beshu.ror.utils
+
+import monix.execution.atomic.Atomic
+
+
```
Comment (coutoPL) [inline core/src/main/scala/tech/beshu/ror/utils/SetOnce.scala:5]: * redundant \n
* no license

File: es79x/src/main/scala/tech/beshu/ror/es/ReadonlyRestPlugin.scala [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+      .map(ssl =>
+        "ror_ssl_internode" -> new Supplier[Transport] {
+          override def get(): Transport = new SSLNetty4InternodeServerTransport(settings, threadPool, pageCacheRecycler, circuitBreakerService, namedWriteableRegistry, networkService, ssl, getSharedGroupFactory(settings))
+        }
+      )
+      .toMap
+      .asJava
+  }
+
+  private val groupFactory = new SetOnce[SharedGroupFactory]
```
Comment (coutoPL) [inline es79x/src/main/scala/tech/beshu/ror/es/ReadonlyRestPlugin.scala:184]: let's move it before methods

**Review by coutoPL — CHANGES_REQUESTED** [review]

File: es79x/src/main/scala/tech/beshu/ror/es/ReadonlyRestPlugin.scala (hunk 1 above)
Comment (clutroth) [reply es79x/src/main/scala/tech/beshu/ror/es/ReadonlyRestPlugin.scala:84]: MapperService moved them to `META_FIELDS_BEFORE_7DOT8`, and all of them, except `IgnoredFieldMapper.NAME` are already in `FIELDS_ALWAYS_ALLOW`

**Review by coutoPL — APPROVED** [review]
LGTM. Thanks

**Pull request merged**